src/services/project/project-detail/project-member/modules/ProjectMemberUpdateModal.vue

```diff
@@ -117,7 +117,7 @@ const tagList = (proxyTags?: Ref<string[]>|null, checkDuplicate = true) => {
      */
     const addTag = (value) => {
         const val = (typeof value === 'string') ? value.trim() : value;
-        if (!val || val === '') return;
+        if (val === '' || !val) return;
```

Comment:
```suggestion
        if (!val) return;
```

`!val` 이 `val === ''` 를 포함해서.. 이렇게 바꾸면 좋을듯요!